src/main/webapp/profile.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Insert cool name here</title>
```

Comment:
Probably should change this

Reply:
Title has been changed to "Your Profile".